.gitignore

```diff
@@ -42,3 +43,10 @@ hyperspy/io_plugins/unbcf_fast.c
 hyperspy/misc/etc/test_compilers.obj
 /.cache/
 /.spyproject/
+
+### Code ###
+# Visual Studio Code - https://code.visualstudio.com/
```

Comment:
what is this?

Reply:
i reset my gitignore to remove a merging conflict